app/src/main/java/com/google/auto/value/AutoValue.java

```diff
@@ -0,0 +1,154 @@
+package com.google.auto.value;
```

Comment:
This was the solution recommended via https://github.com/google/auto/issues/268#issuecomment-143242173 to keep AutoValue working after updating AGP.